.github/workflows/manual-nixos.yml

```diff
@@ -29,3 +29,25 @@ jobs:
           signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
       - name: Building NixOS manual
         run: NIX_PATH=nixpkgs=$(pwd) nix-build --option restrict-eval true nixos/release.nix -A manual.x86_64-linux
+      - name: Compare DocBook and MD manuals
```

Comment:
I don't think this should be added here. 

This is a contributor facing check avoid the manual failing to build and then blocking nixos channels. 

I don't see that comparing docbook and md is relevant to this check, or why it even needs to be a github action?

Reply:
> neither of those need to be done in an action ... and they can still be merged regardless if they're important enough

Then I don't understand why it's being added here. This is intentionally a very simple check, i don't want people to ignore it.

> for either problem to be resolved before they are merged 

What problems? I don't see how anything from this is actionable by a contributor.  "open a issue" means it shouldn't be in this check.

Reply:
> what do you propose we do instead

I don't understand why it needs to be a PR check?

Reply:
> it checks for consistency of the nixpkgs documentation with the proposed policy ("don't use docbook in nixpkgs"), that seems like a pretty reasonable thing to put in an action.

Yes, this seems fine as it would be something that would be problematic in PRs and would be better resolved in the PR before merge.

> the other check ("docbook manual and md manual render the same") extends that to make sure the two options to produce the manual don't produce wildly different results

This doesn't seem like something a casual contributor should ever need to see.

> is there a better place to put that? a nixos test could also check that, but if it's not a channel blocker it'll be routinely forgotten and thus not very useful.

Could have an action on cron that posts a comment like we have for merge branches?